layouts/_default/impact.html

```diff
@@ -0,0 +1,83 @@
+{{ define "main" }}
+  <section class="bg-white">
+    {{ if .Params.pageTitle }}
+      <h1
+        class="mx-auto px-4 pt-16 text-4xl font-bold md:w-7/12 md:px-0 lg:text-5xl"
+      >
+        {{ .Params.pageTitle }}
+      </h1>
+    {{ end }}
+
+
+    <div
+      data-ga-category="content"
+      class="article-content mx-auto w-full px-4 py-6 pb-10 md:w-7/12 md:px-0"
+    >
+      {{ if .Params.intro }}
+        {{ .Params.intro | markdownify }}
+      {{ end }}
+    </div>
+
+    <div id="award-quote-block" class="mx-auto md:w-7/12 md:px-0">
```

Comment:
Avoid using IDs because it means we can't ever repeat something on the page later. A lot of things you think you only need one of end up needing to be repeated. For things that do require IDs, like label-input pairs, we have a template helper:

```
{{ $id := partial "helper/new-id" . }}
<label for="{{ $id }}>
  Something
</label>

<input id="{{ $id }}" />
```

It just automatically increments label-uid-1, label-uid-2, etc. each time you use it on a page, so each call to the partial is unique.